Formula/avfs.rb

```diff
@@ -4,6 +4,11 @@ class Avfs < Formula
   url "https://downloads.sourceforge.net/project/avf/avfs/1.1.4/avfs-1.1.4.tar.bz2"
   sha256 "3a7981af8557f864ae10d4b204c29969588fdb526e117456e8efd54bf8faa12b"
 
+  livecheck do
+    url "https://sourceforge.net/projects/avf/rss"
```

Comment:
```suggestion
    url :stable
```

We can simply use `url :stable` in this case, since we don't need to append the `?path=` query string parameter to the URL in this instance.

Reply:
👍